python/pyspark/streaming/tests.py

```diff
@@ -206,6 +207,22 @@ def func(dstream):
         expected = [[len(x)] for x in input]
         self._test_func(input, func, expected)
 
+    def test_slice(self):
```

Comment:
@holdenk Comments please on __test_slice()__ and on the test results 
https://github.com/apache/spark/pull/20838#issuecomment-408566860